javascript/index.js

```diff
@@ -28,13 +28,119 @@
+        }, (error) => console.log(error));
+      }, (error) => console.log(error));
+    }, (error) => console.log(error));
+  }, (error) => console.log(error));
+}, (error) => console.log(error));
+
+
 
 // Iteration 2 - using promises
-// ...
+
+obtainInstruction('steak', 0)
+  .then(response=> {
+    document.querySelector("#steak").innerHTML += `<li>${response}</li>`;
+  }).then(function() {
+    return obtainInstruction('steak', 1)
+  })
+  .then(response=> {
+    document.querySelector("#steak").innerHTML += `<li>${response}</li>`;
+  })
+  .then(function() {
+    return obtainInstruction('steak', 2)
+  })
+  .then(response=> {
+    document.querySelector("#steak").innerHTML += `<li>${response}</li>`;
+  })
+  .then(function() {
+    return obtainInstruction('steak', 3)
+  })
+  .then(response=> {
+    document.querySelector("#steak").innerHTML += `<li>${response}</li>`;
+  })
+  .then(function() {
+    return obtainInstruction('steak', 4)
+  })
+  .then(response=> {
+    document.querySelector("#steak").innerHTML += `<li>${response}</li>`;
+  })
+  .then(function() {
+    return obtainInstruction('steak', 5)
+  })
+  .then(response=> {
+    document.querySelector("#steak").innerHTML += `<li>${response}</li>`;
+  })
+  .then(function() {
+    return obtainInstruction('steak', 6)
+  })
+  .then(response=> {
+    document.querySelector("#steak").innerHTML += `<li>${response}</li>`;
+  })
+  .then(function() {
+    return obtainInstruction('steak', 7)
+  })
+  .then(response=> {
+    document.querySelector("#steak").innerHTML += `<li>${response}</li>`;
+  })
+  .then(function(){
+    document.querySelector("#steak").innerHTML += `<li>Steak is ready</li>`
+  })
+  .finally(function(){
+    document.querySelector("#steakImg").removeAttribute("hidden")
+  })
+   
+  // ... Your code here
 
 // Iteration 3 using async/await
-// ...
+
+async function makeBroccoli() {
+  let step1 = await obtainInstruction('broccoli', 0)
+  document.querySelector("#broccoli").innerHTML += `<li>${step1}</li>`;
+  let step2 = await obtainInstruction('broccoli', 1)
+  document.querySelector("#broccoli").innerHTML += `<li>${step2}</li>`;
+  let step3 = await obtainInstruction('broccoli', 2)
+  document.querySelector("#broccoli").innerHTML += `<li>${step3}</li>`;
+  let step4 = await obtainInstruction('broccoli', 3)
+  document.querySelector("#broccoli").innerHTML += `<li>${step4}</li>`;
+  let step5 = await obtainInstruction('broccoli', 4)
+  document.querySelector("#broccoli").innerHTML += `<li>${step5}</li>`;
+  let step6 = await obtainInstruction('broccoli', 5)
+  document.querySelector("#broccoli").innerHTML += `<li>${step6}</li>`;
+  let step7 = await obtainInstruction('broccoli', 6)
+  document.querySelector("#broccoli").innerHTML += `<li>${step7}</li>`;
+  document.querySelector("#broccoli").innerHTML += `<li> Broccoli is ready </li>`
+}
+makeBroccoli().then(function() {
+  document.querySelector("#broccoliImg").removeAttribute("hidden")
+})
```

Comment:
Same as above, you can move line 128 after line 125, will have same effect and will be shorter syntax ;)